src/html/html-document.ts

```diff
@@ -63,6 +63,17 @@ export class ParsedHtmlDocument extends ParsedDocument<ASTNode, HtmlVisitor> {
         end: {line: location.endTag.line - 1, column: location.endTag.col - 1}
       };
     }
+    // text node value can provide accurate line and column count
```

Comment:
Will update.  I also just committed fix for comment nodes.  About to send one for elements with comment updates.